<launch>
  <node pkg="point_cloud_mapping" type="pcd_to_msg_node" name="pcd_to_msg_node" output="screen" respawn="true"
	args="$(optenv pcd /home/marton/work/datasets/NICO/lancaster_smooth_trans_rot.pcd) $(optenv time_pcd 2)" />
  
  <node pkg="cloud_algos" type="box_fit2_node" name="box_fit2_node" output="screen" respawn="true">
    <remap from="~cloud_pcd" to="cloud_pcd" />
    <param  name="output_box_topic" type="string" value="box_marker"/>
    <param  name="threshold_in" type="double" value="0.02"/>
    <param  name="threshold_out" type="double" value="0.00001"/>
    <param  name="eps_angle" type="double" value="0.1"/>
    <param  name="success_probability_" type="double" value="1"/>
  </node>
</launch>
